.pre-commit-config.yaml

```diff
@@ -3,13 +3,23 @@ default_language_version:
 
```

Comment:
Precommit was blowing up on me because the previous flake8 config wasn't working any more. So I copied over the more recent precommit config from the cdk handler repo.